### PR TITLE
Adding Russian translation #13

### PR DESCRIPTION
**Pull request by sashman**

### CODE REVIEW

**Comment by sashman** [comment]
Just doing some testing...

**Comment by sashman** [comment]
Should be working, however, the Russian name cases for month/day of week names don't change...

This might be a due to the mechanism of swapping the names in

File: humanized_opening_hours/locales/ru_RU/LC_MESSAGES/hoh.pot [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-10-06 14:06+0100\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
```
Comment (rezemika) [inline humanized_opening_hours/locales/ru_RU/LC_MESSAGES/hoh.pot]: Thank you! Can you just change this line to `"Language: ru_RU\n" please? :)

**Comment by rezemika** [comment]
It's squashed, thank you very much! I'll see if I can make this rendering more flexible. :)

**Pull request merged**